From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Wed, 18 May 2022 17:48:16 +0800
Subject: demos: Fix distcheck

Make sure that we remove the files generated by glib-compile-resources
during distclean.

Origin: upstream, 3.24.35, commit:0264e2d6a6cf748f6adbcfbe06b85fe184774c2e
---
 demos/gtk-demo/Makefile.am       | 2 +-
 demos/icon-browser/Makefile.am   | 2 ++
 demos/widget-factory/Makefile.am | 2 ++
 3 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/demos/gtk-demo/Makefile.am b/demos/gtk-demo/Makefile.am
index b0ecbcc..0616798 100644
--- a/demos/gtk-demo/Makefile.am
+++ b/demos/gtk-demo/Makefile.am
@@ -129,6 +129,6 @@ dist-hook: \
 	$(top_builddir)/win32/vs9/gtk3-demo.vcproj	\
 	$(top_builddir)/win32/vs9/gtk3-demo-application.vcproj
 
-DISTCLEANFILES = demos.h demos.h.win32
+DISTCLEANFILES = demos.h demos.h.win32 demo_resources.c
 
 -include $(top_srcdir)/git.mk
diff --git a/demos/icon-browser/Makefile.am b/demos/icon-browser/Makefile.am
index 4ba64ee..9183b01 100644
--- a/demos/icon-browser/Makefile.am
+++ b/demos/icon-browser/Makefile.am
@@ -27,6 +27,8 @@ gtk3_icon_browser_SOURCES = \
 BUILT_SOURCES = \
 	resources.c
 
+DISTCLEANFILES = resources.c
+
 resources.c: iconbrowser.gresource.xml $(shell $(GLIB_COMPILE_RESOURCES) --sourcedir=$(srcdir) --generate-dependencies $(srcdir)/iconbrowser.gresource.xml)
 	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) $(srcdir)/iconbrowser.gresource.xml \
 		--target=$@ --sourcedir=$(srcdir) --generate-source
diff --git a/demos/widget-factory/Makefile.am b/demos/widget-factory/Makefile.am
index 1afbb50..5faacb0 100644
--- a/demos/widget-factory/Makefile.am
+++ b/demos/widget-factory/Makefile.am
@@ -69,4 +69,6 @@ EXTRA_DIST += 				\
 	data/symbolic-source.svg	\
 	meson.build
 
+DISTCLEANFILES = widget_factory_resources.c
+
 -include $(top_srcdir)/git.mk
